File: python class 1/factorial.py
def factorial(n):
    if n == 0 or n == 1:
        return 1
    else:
        result = 1
        for i in range(2, n+1):
            temp = result
            for j in range(1, i):
                temp += result
            result = temp
        return result

num = int(input("Ingresa un número: "))
print(factorial(num))

File: python class 1/factorial.js
function factorial(n) {
    if (n === 0 || n === 1) {
        return 1;
    } else {
        var result = 1;
        for (var i = 2; i <= n; i++) {
            var temp = result;
            for (var j = 1; j < i; j++) {
                temp += result;
            }
            result = temp;
        }
        return result;
    }
}
var num = process.argv[2];
console.log(factorial(num));
